projectroles/forms.py

```diff
@@ -957,6 +957,24 @@ def clean(self):
         except ProjectInvite.DoesNotExist:
             pass
 
+        # Local users check
+        user_email = self.cleaned_data.get('email')
+        domain = user_email[user_email.find('@') + 1 :]
```

Comment:
When comparing against `AUTH_LDAP*_USERNAME_DOMAIN`, note that the expected values here don't include the top level domain. Hence we should be comparing them to the email domain without the TLD. See more in the comment below.